frontend/src/app/testResults/viewResults/actionMenuModals/TestResultDetailsModal.tsx

```diff
@@ -24,33 +27,6 @@ import "./TestResultPrintModal.scss";
 import { MULTIPLEX_DISEASES } from "../../constants";
 import { formatGenderOfSexualPartnersForDisplay } from "../../../utils/gender";
 
-type Result = {
```

Comment:
⚠️ Removing this loose type in favor of getting the type directly from the query below.

```
type QueriedTestResult = NonNullable<GetTestResultDetailsQuery>["testResult"];
```

Reply:
ooo nice! I prefer getting the types directly than defining/maintaining our own again!